vite-project/src/components/listItemWithBorder.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+const ListItemWithBorder = ({ children }) => {
+    return (
+      <li style={{border: "1px solid rgb(3, 0, 47)",
```

Comment:
please do not use inline styling until its necessary. Move it to CSS file please :)